File: GrowthWare_VB/GrowthWareFrameworkModel/Profiles/MWorkFlowProfileCollection.vb
Imports System.Runtime.Serialization
Imports GrowthWare.Framework.ModelObjects.Base

Namespace ModelObjects
	<Serializable(), CLSCompliant(True)>
	Public NotInheritable Class MWorkFlowProfileCollection
		Inherits MProfileCollection

		Protected Sub New(ByVal info As SerializationInfo, ByVal context As StreamingContext)
			MyBase.New(info, context)
		End Sub
	End Class
End Namespace

File: GrowthWare_CS/GrowthWareFrameworkModel/Profiles/MWorkFlowProfileCollection.cs
using System;
using System.Runtime.Serialization;
using GrowthWare.Framework.ModelObjects.Base;

namespace GrowthWare.Framework.ModelObjects
{
	[Serializable(), CLSCompliant(true)]
	public sealed class MWorkFlowProfileCollection : MProfileCollection
	{

		MWorkFlowProfileCollection(SerializationInfo info, StreamingContext context)
			: base(info, context)
		{
			// do nothing
		}
	}
}
